easybuild/framework/easyblock.py

```diff
@@ -2667,6 +2678,18 @@ def build_and_install_one(ecdict, init_env, hooks=None):
         _log.debug("Stop set to %s" % stop)
         app.cfg['stop'] = stop
 
+    continue_from = build_option('continue')
+    if continue_from is not None:
+        _log.experimental("Continuing from step %s" % continue_from)
+        skipsteps = []
+        for step in build_option('valid_stops'):
```

Comment:
I haven't checked, but is build_option('valid_stops') guaranteed to be in the correct order?

Reply:
Yes, it will be in the correct execution order. The list in `valid_stops` is in the same order as `get_steps`.

Maybe it's worth the trouble to add an explicit test for this if we really on that, doesn't hurt...